src/js/grapl-cdk/lib/grapl-cdk-stack.ts

```diff
@@ -709,76 +710,10 @@ export class ModelPluginDeployer extends cdk.NestedStack {
             props.modelPluginBucket.grantDelete(event_handler.role);
         }
 
-        const integration = new apigateway.LambdaRestApi(this, 'Integration', {
-            restApiName: this.integrationName,
-            endpointExportName: serviceName + '-EndpointApi',
-            handler: event_handler,
+        const integration = new apigateway.LambdaIntegration(event_handler);
+        props.edgeApi.root.addResource('modelPluginDeployer').addProxy({
+            defaultIntegration: integration,
         });
-
-        integration.addUsagePlan('integrationApiUsagePlan', {
-            quota: {
-                limit: 1000,
-                period: apigateway.Period.DAY,
-            },
-            throttle: {
-                // per minute
-                rateLimit: 50,
-                burstLimit: 50,
-            },
-        });
-
-        if (props.watchful) {
-            props.watchful.watchApiGateway(
```

Comment:
Did we lose per-path HTTP metrics in this PR? If so, how can we get them back?

Reply:
I'm actually not sure. I would sort of assume that watchful just does that, but I'll check. If not, I can try to take a quick pass to fix this.

Reply:
Yeah, we did regress here, fixing now.